lib/tasks/create_test_accounts.rb

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env ruby
+
+# Create test user accounts
+#
+# This script includes functions that can be used to create test accounts
+# locally and in our integration environments.
+
+
+# Creates a user account with given attributes
+def create_account(email: 'joe.smith@email.com', password: 'salty pickles', mfa_phone: '1234567890', verified: true,
+                   first_name: 'joe', middle_name: 'jingles', last_name: 'smith', dob: '1/1/1970', ssn: '123456789',
+                   address1: '123 America St', address2: 'Apt 1776', city: 'Washington', state: 'DC', zipcode: '20001', phone: '9876543210')
+
+  email.downcase!
+  ee = EncryptedAttribute.new_from_decrypted(email)
+  User.find_or_create_by!(email_fingerprint: ee.fingerprint) do |user|
+    user.encrypted_email = ee.encrypted
+    user.skip_confirmation!
+    user.reset_password(password, password)
+    user.phone = mfa_phone || phone
+    user.phone_confirmed_at = Time.zone.now
+    Event.create(user_id: user.id, event_type: :account_created)
+  end
```

Comment:
For this script to work with KMS, we can't create a user by a fingerprint generated `EncryptedAttribute.new_from_decrypted(email)`. The way I got it to work was to replace lines 15-23 with this:
```ruby
user = User.create!(email: email)
user.skip_confirmation!
user.reset_password(password, password)
user.phone = mfa_phone || phone
user.phone_confirmed_at = Time.zone.now
user.save!
Event.create(user_id: user.id, event_type: :account_created)
```
And then to skip existing users, I added this between lines 63 and 64:
```ruby
email = row['email']
if User.find_with_email(email).present?
  Rails.logger.warn "user with email #{email} already exists"
  next
end
```

Reply:
@blacktm Did you still want this script to be merged? If so, wanna make the changes I proposed?

Reply:
Yes, will do, thanks for the reminder.